docs/source/osds/scenarios.rst

```diff
@@ -0,0 +1,26 @@
+OSD Scenarios
+=============
+
+lvm
+---
+This OSD scenario uses ``ceph-volume`` to create OSDs from logical volumes and
+is only available when the ceph release is Luminous or greater.
+
+.. note::
+   The creation of the logical volumes is not supported by ceph-ansible, ceph-volume
```

Comment:
``` ``ceph-volume`` ``` and  ``` ``ceph-ansible`` ``` (with double ticks all the time) so that it is consistent everywhere it is used